### DISCUSSION TITLE
How to remove the date folder from output?

### DISCUSSION THREAD

**Discussion by Teedo145** [lllyasviel/Fooocus#1529] [Q&A]
I want all outputs to be in a single folder rather than sorted by date, is this possible?

**Comment by juliux32** [comment]
But why? 🫤 it's much easier to have a folder from yesterday and know what you did that day, rather than having one folder with possibly hundreds if not thousands of pictures mixed in one folder

**Reply by Teedo145** [reply]
Because i'll be moving them to my own folders...but it's alright, i can just use the link the result response

**Reply by juliux32** [reply]
If you are sorting your creations to specific folders, then there's no difference if your images go to the **"all"** folder or **"2023-12-21"** 🫤 because when you will end your session, you will be moving images from the last folder... or I am missing something? 

MAYBE if you are sorting your images once a week or month, then it could be a little more job to check every folder, BUT STILL... you would have more control because you could mark for yourself **"ok, I have already sorted all the images from December 1-10, so tomorrow I will check images from December 11-15"**

**Reply by Teedo145** [reply]
The sorting is done automatically and i didn't want to write code to check the correct folder by date on the latest images to sort. I figured sorting it by dates was a simple option in a cfg somewhere. But I've done it slightly different now and the date sorting doesn't really matter now as the generation prints a direct link to the image file.
